shell/platform/windows/flutter_windows_view.cc

```diff
@@ -10,14 +10,15 @@ namespace flutter {
 
 FlutterWindowsView::FlutterWindowsView(
     std::unique_ptr<WindowBindingHandler> window_binding) {
-  surface_manager_ = std::make_unique<AngleSurfaceManager>();
-
   // Take the binding handler, and give it a pointer back to self.
```

Comment:
This comment in part describes the line you moved; if the new ordering is important please update this comment and add a comment to the various lines where order matters explaining why. (If not, keeping them grouped would be preferable.)

Reply:
Resolved in next commit